src/style-spec/reference/v8.json

```diff
@@ -861,6 +861,9 @@
         },
         "line": {
           "doc": "The label is placed along the line of the geometry. Can only be used on `LineString` and `Polygon` geometries."
+        },
+        "line-center": {
```

Comment:
Can we add more specific compatibility info? See
https://github.com/mapbox/mapbox-gl-js/blob/a8cb4c48fb66868ccca00309da30518cf7637eef/src/style-spec/reference/v8.json#L1035-L1040 for example.

Reply:
Good idea -- this look right?

<img width="667" alt="screenshot 2018-07-03 14 00 05" src="https://user-images.githubusercontent.com/375121/42244325-720d3802-7ec9-11e8-991f-e867da3d91c4.png">